primefaces-showcase/src/main/webapp/accessibility.xhtml

```diff
@@ -1,8 +1,233 @@
+                to better user experience for people without disabilities so designing for cognitive impairments result in better design for any user.
+            </p>
+
+            <h3>Web Content</h3>
+            <p>Correct page structure with the aid of assistive technologies are the core ingridients for an accessible web content.
+                HTML is based on an accessible foundation, form controls can be used by keyboard by default and semantic HTML is easier to be processed by a screen reader.</p>
+
+            <h5>WCAG</h5>
+            <p><a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">WCAG</a> refers to <strong>Web Content Accessibility Guideline</strong>, a standard managed by the WAI (Web Accessibility Initiative) of W3C (World Wide Web Consortium).
+                WCAG consists of recommendations for making the web content more accessible. PrimeFaces components aim high level of WCAG compliancy in the near future.</p>
+
+            <p>Various countries around the globe have governmental policies regarding web accessibility as well. Most well known of these are <a href="https://www.section508.gov/manage/laws-and-policies/" target="_blank">Section 508</a> in the US and <a href="https://digital-strategy.ec.europa.eu/en/policies/web-accessibility" target="_blank">Web Accessibility Directive</a> of the European Union.</p>
+
+            <h5>Form Controls</h5>
+            <p>Native form elements should be preferred instead of elements that are meant for other purposes like presentation. As an example, button below is
+                rendered as a form control by the browser, can receive focus via tabbing and can be used with space key as well to trigger.
+            </p>
+            <h:form rendered="false">
+                <p:messages />
+                <h:commandButton value="Click Button" action="#{accessiblityView.someAction()}">
+                    <f:ajax execute="@form" render="@form" />
+                </h:commandButton>
+                <br/>
+                <p:remoteCommand name="rc" update="@form" action="#{accessiblityView.someOtherAction()}"/>
+                <div class="fancy-button" onclick="rc()">Click div</div>
+                <div class="fancy-button" onclick="rc()" onkeydown="rc()" tabindex="0">Click div improved</div>
+            </h:form>
+            <pre><code class="language-xml">
+                <h:outputText value="#{accessiblityView.htmlFragmentButton}" />
+            </code></pre>
+
+            <p>On the other hand, a fancy css based button using a div has no keyboard or screen reader support.</p>
+            <pre><code class="language-xml">
+                <h:outputText value="#{accessiblityView.htmlFragmentDivAsButton}" />
+            </code></pre>
+
+            <p><i>tabindex</i> is required to make a div element accessible in addition to use a keydown to bring the keyboard support back. To avoid the overload and implementing
```

Comment:
`<code>` is more semantic here.
```suggestion
            <p><code>tabindex</code> is required to make a div element accessible in addition to use a keydown to bring the keyboard support back. To avoid the overload and implementing
```